peer/store-utils.go

```diff
@@ -11,12 +11,21 @@ import (
 	"github.com/gluster/glusterd2/store"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/pborman/uuid"
 )
 
 const (
 	peerPrefix string = store.GlusterPrefix + "peers/"
 )
 
+var (
```

Comment:
Ah ok. Although it seems like a bad practice to change actual code to fix unit tests.